antora-ui-camel/src/partials/BackToTop.hbs

```diff
@@ -0,0 +1 @@
+<div class="top"><a href="#top">Back To Top </a></div>
```

Comment:
Not sure we need a partial for this, it's quite small to begin with. If it is needed then it needs to conform to the file naming conventions other partials have.

Reply:
> I'd prefer to put the back to top link in the same place on the Antora built content and as on the Hugo built content.
@zregvart  Actually earlier, I was going with the same thing as you are suggesting, but I encountered a problem as shown below in the documentation pages. I thought maybe that's the reason Edit Page link is also not set at the bottom in the Documentation pages. 


![Screenshot from 2020-04-01 19-15-11](https://user-images.githubusercontent.com/51945896/78144385-27f6a200-744d-11ea-9f29-0580d990f616.png)

Reply:
> Not sure we need a partial for this, it's quite small to begin with. If it is needed then it needs to conform to the file naming conventions other partials have.


Made the changes